vendor/bat-native-ads/src/bat/ads/internal/common/search_engine/search_engine_subdomain_constants.cc

```diff
@@ -11,7 +11,7 @@ namespace ads {
 
 const std::vector<std::string>& GetWikipediaSearchEngineSubdomains() {
   // See https://www.wikipedia.org/.
-  static base::NoDestructor<std::vector<std::string>> extensions(
+  static const base::NoDestructor<std::vector<std::string>> kExtensions(
```

Comment:
This should be converted to `static constexpr char* kExtensions[]`

Reply:
agree. no need to all these static arrays use runtime memory.

Reply:
@aseren will fix in his related PR